docs/user/quick-start/first-steps.md

```diff
@@ -0,0 +1,62 @@
+---
+sidebar_position: 1
+---
+
+# First steps with DAppNode
+---
+If you bought a DAppNode from our [store](https://shop.dappnode.io/), built your own custom NUC or installed DAppNode on a device you have lying around, theres a few steps you need to do in order start accessing your favorite chains.  
+
+
+### First login
+
+The first time you access the [DAppNode UI](http://my.dappnode), you will be asked to create a new account on your DAppNode. You will need to supply a username and a password. We use the following password:
```

Comment:
```suggestion
The first time you access the [DAppNode UI](http://my.dappnode), you will be asked to create a new account on your DAppNode. You will need to supply a username and a strong password. We have the following password requirements:
```

Im not sure if this is what you meant but what it says doesnt make sense

Reply:
Yup, thats what I intended to write there, thanks for catching that!